src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,28 @@
+package racing.domain;
+/**
+ * 자동차
+ * 1. 전진 & 멈추기
+ * 2. 현재 위치 반환
+ * */
+public class Car {
+  private int position;
+  public Car(int position) {
+    this.position = position;
+  }
+
+  public boolean move(int value){
```

Comment:
```suggestion
  public void move(int value){
```
굳이 boolean 값을 반환할 필요가 있을까?
tdd에서 값 검증이 필요하면 position 이동 여부로 판단하면 되지 않을까?